java/pom.xml

```diff
@@ -122,6 +122,11 @@
     		<version>2.0.27</version>
 		</dependency>
 		<dependency>
+    		<groupId>org.jsoup</groupId>
+    		<artifactId>jsoup</artifactId>
```

Comment:
@tomas-sexenian as we added a new required dependency, do you know how mucho size this JAR takes?

Reply:
Jsoup 1.16 takes 433 KB of space which I agree is not optimal. Advantages are that it is open source, free for comercial use, has no known vulnerabilities and its complete enough to handle more complex HTML structures if, in the future, we wanted to expand HTML rendering for PDFBox generated reports.

I can try rewriting what I have now but using another HTML parsing library if it is necessary